src/dev/register_git_hook/register_git_hook.js

```diff
@@ -111,7 +111,15 @@ function getKbnPrecommitGitHookScript(rootPath, nodeHome, platform) {
     exit 1
   }
 
-  node scripts/precommit_hook || {
+  execute_local_precommit_hook() {
+    PRECOMMIT_FILE=".pre-commit"
+    if [ -f \${PRECOMMIT_FILE} ]; then 
+      echo "Executing local precommit hook found in \${PRECOMMIT_FILE}" 
+      sh \${PRECOMMIT_FILE}
+    fi
+  }
+
+  node scripts/precommit_hook || execute_local_precommit_hook {
```

Comment:
Lets rename `execute_local_precommit_hook` to `execute_precommit_hook` and call `node scripts/precommit_hook` from the top of that function, I'm pretty sure the `||` in this line is wrong and I'm not sure how `&&` and `||` ordering works in bash...

Reply:
ah yeah, I made those changes in the actual precommit hook and forgot to put it back into the template. thanks, good catch